plugins/microsoft_atp/Dockerfile

```diff
@@ -1,36 +1,20 @@
-FROM rapid7/insightconnect-python-3-38-slim-plugin:4
-# Refer to the following documentation for available SDK parent images: https://docs.rapid7.com/insightconnect/sdk-guide/#sdk-guide
+FROM rapid7/insightconnect-python-3-38-plugin:5
 
 LABEL organization=rapid7
 LABEL sdk=python
 
-# Add any custom package dependencies here
-# NOTE: Add pip packages to requirements.txt
-
-# Need gcc for maya
-RUN apk add --no-cache --virtual .build-deps \
-                                        make \
-                                        gcc \
-                                        libc-dev \
-                                        libffi-dev \
-                                        openssl-dev \
-                                        libxml2-dev \
-                                        libxslt-dev
```

Comment:
Is that necessary to be removed? If its required for a plugin to run try add the following to the `sdk` in `plugin.spec.yaml`

```
sdk:
  ...
  packages:
    - make
    - gcc
    etc...
```

Reply:
This was removed when the refresh tool was run, however when I make the new docker image using this docker file and exec the container we can see that that maya is still installed and that the actions work as expected

```
docker exec -it atp /bin/sh

$ pip list | grep maya
WARNING: The directory '/nonexistent/.cache/pip' or its parent directory is not owned or is not writable by the current user. The cache has been disabled. Check the permissions and owner of that directory. If executing pip with sudo, you may want sudo's -H flag.
maya                          0.5.0
```

Reply:
Thats fine 😄